acrobat/scripts/scripts.js

```diff
@@ -455,6 +455,12 @@ const { ietf } = getLocale(locales);
   const { loadArea, setConfig, loadLana, getMetadata, loadIms } = await import(`${miloLibs}/utils/utils.js`);
   addLocale(ietf);
 
+  // IMS Ready
+  loadIms().then(() => {
```

Comment:
do we need to call `setConfig` before `loadIMS`?  does loadIMS rely on the client id, etc?